### ISSUE TITLE
leafcutter_DS halted 

### ISSUE THREAD

**Issue by herber4** [davidaknowles/leafcutter#198]
Loading required package: Rcpp
Loading counts from Book1.txt 
Loading metadata from bam.list 
Error in `[.data.frame`(counts, , meta$sample) : 
  undefined columns selected
Calls: [ -> [.data.frame
Execution halted

any idea why this would happen? My input files are the same as used in the tutorial

**Comment by jaempawi** [comment]
I'm having the same issue. Any update?

**Comment by herber4** [comment]
yes, make sure the sample names in the meta data file are the same as what's in the counts table. in the case of the example above, you can:
cat Book1.txt
cat bam.list

then edit the names in the bam.list to match the sample names in the Book1.txt
